.github/workflows/build_pr_documentation.yml

```diff
@@ -54,15 +54,15 @@ jobs:
         run: |
           cd optimum-habana/habana-doc-build
           sudo mv optimum.habana optimum-habana
-          doc-builder push optimum-habana --doc_build_repo_id "huggingface/doc-build-dev" --token ${{ env.write_token }} --commit_msg "Updated with commit $COMMIT_SHA See: https://github.com/huggingface/optimum-habana/commit/$COMMIT_SHA" --n_retries 5
+          doc-builder push optimum-habana --doc_build_repo_id "hf-doc-build/doc-build-dev" --token "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" --commit_msg "Updated with commit $COMMIT_SHA See: https://github.com/huggingface/optimum-habana/commit/$COMMIT_SHA" --n_retries 5
```

Comment:
Is it okay to have the token available in plain text?

Reply:
yes, because we need those tokens for the forks to be able to run github actions & the tokens are scoped